changelog.txt

```diff
@@ -1,10 +1,21 @@
+---------------------------------------------------------------------------
+
+Version 2.11.0, August 22, 2019
+
+The Checker Framework uses the Error Prone version of javac.  Compared to
```

Comment:
How about:

```
The Checker Framework now uses the Java 9 javac API. The Error Prone javac can be used
as a simple dependency that works on a Java 8 JVM.
```

The Checker Framework depends on the normal Java 9 javac API. We don't depend on any Error Prone extensions.
People can simply use the normal Java 9+ javac (modulo the annotated JDK).
However, the Error Prone javac is an easily available Maven dependency that continues to work on a Java 8 JVM.

Error Prone is a version of the Java 9 javac, not of the Java 8 javac.
It does contain a few bugfixes compared to the standard javac, but we don't depend on any of them in particular.

The second sentence from the changelog also appears twice below. Those sentences should similarly be updated.

Reply:
Aren't there bug fixes that correct problems in the generated .class files?  I seem to recall problems with those incorrect .class files.

Reply:
I couldn't find a list of differences between OpenJDK 9 javac and Error Prone javac.
All framework tests work with normal OpenJDK 9 to 11 javac, so if there are any differences we're not testing for them.